syncstorage/src/web/extractors.rs

```diff
@@ -112,6 +117,17 @@ impl BatchBsoBody {
     }
 }
 
+impl From<BatchBsoBody> for PostCollectionBso {
```

Comment:
Leave this for now but I'll note we had a TODO comment somewhere that dissapeared to kill one of these types. I logged https://github.com/mozilla-services/syncstorage-rs/issues/1296 to revisit it later.